master/buildbot/newsfragments/multifileupload_glob.feature

```diff
@@ -0,0 +1 @@
+:py:class: `~buildbot.steps.transfer.MultipleFilUpload` now supports the glob parameter. If `glob` is set to `True` all workersrcs parameters will be run through `glob` and the result will be uploaded to `masterdest`
```

Comment:
I believe there shouldn't be space after `:py:class:`.
Use double backticks instead single one:
```
``glob``
```
Dot is missing at the end of second sentence.